addons/ingress-nginx/resources/daemonset.yml.erb

```diff
@@ -43,6 +33,14 @@ spec:
             - --tcp-services-configmap=$(POD_NAMESPACE)/tcp-services
             - --udp-services-configmap=$(POD_NAMESPACE)/udp-services
             - --annotations-prefix=nginx.ingress.kubernetes.io
+          securityContext:
```

Comment:
This change is from `0.16.0`, see: https://github.com/kubernetes/ingress-nginx/releases/tag/nginx-0.16.0